.github/workflows/automerge.yml

```diff
@@ -0,0 +1,33 @@
+name: automerge
```

Comment:
@sbidoul This workflow definition lets us just add an `automerge` label to the PR and let it be automatically merged when all required CI checks pass.

I don't know if oca git bot is applicable for this repo, given is different nature. That's why I added this, plus it's something I almost do everywhere these days.

If it is redundant with our git bot and you think it would confuse or would be not of any help, I can remove this. You're the pro there, just tell me please.

Reply:
> If it is redundant with our git bot and you think it would confuse or would be not of any help

A bit of all this, yes ;) I think ocabot merge nobump will work just fine here too.